src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/Native/RefreshContainer.Android.cs

```diff
@@ -13,9 +13,7 @@ namespace Microsoft.UI.Xaml.Controls;
 
 public partial class RefreshContainer : ContentControl
 {
-	private const int IndicatorHeight = 64;
 	private readonly SerialDisposable _refreshSubscription = new SerialDisposable();
-	private readonly SerialDisposable _nativeScrollViewAttachment = new SerialDisposable();
```

Comment:
Let's comment out, it's coming from the original code.

Reply:
Is this the original code? https://github.com/microsoft/microsoft-ui-xaml/tree/main/dev/PullToRefresh/RefreshContainer

I can't find those their.

Reply:
Indeed, these are not coming from original code, this was Android specific, now unsued.